packages/stems/src/components/TokenValueInput/TokenValueInput.tsx

```diff
@@ -27,6 +27,9 @@ const formatValue = (value: string | undefined) => {
 
 const inputPaddingRight = 16
 
+/**
+ * @deprecated Use TokenValueInputV2 instead
+ */
```

Comment:
nit, how do we feel about naming the new one TokenValueInput and renaming the old one instead?

Reply:
What if I name the new one `TokenAmountInput` so there's no breaking change but it's still nicely named?

Reply:
👍 like that!